clients/logger/logger.go

```diff
@@ -69,11 +69,6 @@ type fileWriter struct {
 // NewClient creates an instance of LoggingClient
 func NewClient(owningServiceName string, isRemote bool, logTarget string, logLevel string) LoggingClient {
 	lc := newClient(owningServiceName, isRemote, logTarget, logLevel)
-
-	if logTarget == "" {
-		lc.Error("logTarget cannot be blank, using stdout only")
```

Comment:
instead of removing it, should we just change the logging level to debug?

Reply:
Nope, this is a valid configuration and will soon be the only configuration.

Reply:
got it, good to know!